ioctl/newcmd/account/accountimport.go

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"fmt"
+	"github.com/iotexproject/iotex-core/ioctl"
+	"os"
+	"sigs.k8s.io/yaml"
+
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/ioctl/validator"
+)
+
+// Multi-language support
+var (
+	importCmdShorts = map[config.Language]string{
+		config.English: "Import IoTeX private key or keystore into wallet",
+		config.Chinese: "将IoTeX的私钥或私钥库导入钱包",
+	}
+	importCmdUses = map[config.Language]string{
+		config.English: "import",
+		config.Chinese: "导入",
+	}
+	importKeyCmdShorts = map[config.Language]string{
+		config.English: "Import IoTeX private key into wallet",
+		config.Chinese: "将IoTeX的私钥导入钱包",
+	}
+	importKeyCmdUses = map[config.Language]string{
+		config.English: "key ALIAS",
+		config.Chinese: "key 别名",
+	}
+	importKeyStoreCmdShorts = map[config.Language]string{
+		config.English: "Import IoTeX keystore into wallet",
+		config.Chinese: "将IoTeX的私钥库导入钱包",
+	}
+	importKeyStoreCmdUses = map[config.Language]string{
+		config.English: "keystore ALIAS FILEPATH",
+		config.Chinese: "keystore 别名 文件路径",
+	}
+	importPemCmdShorts = map[config.Language]string{
+		config.English: "Import IoTeX key from pem file into wallet",
+		config.Chinese: "将IoTeX私钥从pem文件导入钱包",
+	}
+	importPemCmdUses = map[config.Language]string{
+		config.English: "pem ALIAS FILEPATH",
+		config.Chinese: "pem 别名 文件路径",
+	}
+)
+
+func NewAccountImportKeyCmd(client ioctl.Client) *cobra.Command {
+	importKeyUses, _ := client.SelectTranslation(importKeyCmdUses)
+	importKeyShorts, _ := client.SelectTranslation(importKeyCmdShorts)
+	accountImportKeyCmd := &cobra.Command{
+		Use:   importKeyUses,
+		Short: importKeyShorts,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			err := accountImportKey(client, cmd, args)
+			return output.PrintError(err)
```

Comment:
`return accountImportKey(client, cmd, args)`
what's the value of this private function?